urbanaccess/gtfs/utils_format.py

```diff
@@ -441,15 +457,25 @@ def add_unique_agencyid(agency_df=None,stops_df=None,routes_df=None,trips_df=Non
                                                 trips_df=trips_df[['trip_id', 'route_id']],
                                                 agency_df=agency_df[['agency_id','agency_name']])
 
+            # TODO: It's obfuscatory to update the dataframe variables in such a deeply nested way.
+            #       Perhaps a more clear naming convention here before df_list is overridden
+            #       would make this steps intent more explicit.
+
+            # need to update the df_list object with these new variable overrides
+            df_list = [stops_df,routes_df,trips_df,stop_times_df,calendar_df,calendar_dates_df]
+            
             log('agency.txt agency_name column has more than one agency name listed. Unique agency id was assigned using the agency id and associated agency name.')
 
-    for df in df_list:
+    for index, df in enumerate(df_list):
         if df['unique_agency_id'].isnull().values.any():
+            # TODO: These string conversions seem to follow a pattern, could be part of the helper function?
```

Comment:
This TODO refers to a specific unique_agency_id generation that relies on the feed folder name, not a dataframe column name. This is separate from the column agency name function you created below. I am ok with having this outside a helper function but if you wanted to include this in your function below that is ok, the scope would just increase.